introduction/if-else.py

```diff
@@ -0,0 +1,21 @@
+# Python If-Else
+"""Task
+Given an integer, n, perform the following conditional actions:
+
+If n is odd, print Weird
+If n is even and in the inclusive range of 2 to 5, print Not Weird
+If n is even and in the inclusive range of 6 to 20, print Weird
+If n is even and greater than 20, print Not Weird
+"""
+
+if __name__ == '__main__':
+    n = int(input().strip())
+    if n % 2 == 0:
+        if n in range(2, 6):
```

Comment:
that's an expensive one!
you don't need to create a collection to say whether n belongs to some range of values(hint: n > 1...)